lib/PDF/API2/Annotation.pm

```diff
@@ -193,12 +195,17 @@ sub pdf {
     $self->{'A'}->{'F'} = PDFStr($file);
 
     unless (%options) {
-        my $destination = PDFNum($page_number);
-        $self->{'A'}->{'D'} = _destination($destination, $location, @args);
```

Comment:
Would you change `$destination` to `$page_number` so that we don't have `$dest` and `$destination` in the same scope, please?

Reply:
I can, but these lines are not part of my PR. Is this a case of "if we are changing things here, we might as well change this"?

Reply:
Ok.